lte/gateway/python/magma/pipelined/gtp_stats_collector.py

```diff
@@ -0,0 +1,138 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+
+import asyncio
+from typing import List
+
+import re
+from collections import namedtuple
+
+from magma.common.job import Job
+from magma.magmad.check import subprocess_workflow
+
+from magma.pipelined.metrics import GTP_PORT_USER_PLANE_DL_BYTES, \
+    GTP_PORT_USER_PLANE_UL_BYTES
+
+OVSDBDumpCommandParams = namedtuple('OVSDBCommandParams', ['table', 'columns'])
+ParsedInterfaceStats = namedtuple('ParsedInterfaceStats', [
+    'Interface',
+    'rx_bytes',
+    'tx_bytes',
+    'remote_ip',
+])
+OVSDBCommandResult = namedtuple('OVSDBCommandResult', ['out', 'err'])
+
+interface_tx_rx_stats_re = re.compile(
+    r'"(?P<Interface>\w+)"(.*)(remote_ip="(?P<remote_ip>.*)"(.*))?rx_bytes=(?P<rx_bytes>\d).+tx_bytes=(?P<tx_bytes>\d)')
```

Comment:
for maintainability, can you build this up from a couple of intermediate strings, e.g.

```
iface_match = '"(?P<Interface>\w+)"'
remote_ip_match = '(remote_ip="(?P<remote_ip>)"'
# etc

final_re = re.compile(f'{iface_match}(.*){remote_ip_match}(.*)...')
```